documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,92 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prometheus',
+        rules: [
+          {
+            alert: 'PromScrapeFailed',
+            expr: |||
+              up != 1
+            |||,
+            'for': '15m',
+            labels: {
+              severity: 'warning',
```

Comment:
Ticket and page are our standard severities.

Reply:
hmm .. looking at all the sets of alerts out there my feeling is that `warning` / `critical` is more commonly used. I remember we've had discussions and did not really come to an agreement. Maybe these should be configurable and default to one or the other?

Reply:
Looking through our existing docs, "page" is what we use. No other values are present.

Reply:
There are good arguments in both directions; I'm not super opinionated about this.

But lets not use this PR to discuss this particular issue as it can be divisive; either link to previous discussion where this was decided, start a new discussion on the -dev list, or use the dev summit. 

I'm happy to hold this PR until this is decided.

Reply:
I can't seem to find it among my email. As it stands our existing usage is "page" in the docs, so if that's to be changed that would need discussion.

Reply:
We discussed at the dev summit and settled on critical and warning:

https://docs.google.com/document/d/1-C5PycocOZEVIPrmM1hn8fBelShqtqiAmFptoG4yK70/edit